python/paddle/distributed/auto_parallel/interface.py

```diff
@@ -209,7 +215,9 @@ def __call__(self, *args, **kwargs):
 
             for idx in range(op_size, new_op_size):
                 op = cur_block.ops[idx]
-                op._set_attr("is_recompute@auto_parallel", True)
+                op._set_attr(
```

Comment:
Use another attr like dist attr for this information.
op_namescope is supposed to be used by many other logic. 
if the modified op is not under fully control by auto parallel(like the allreduce only added and used by auto parallel logic),  you might remove the information of  other.

Reply:
Ok, the attribute will be moved to dist_attr in the next PR.